src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -299,6 +304,24 @@ public void SetCheckpointDir(string directory)
             _jvmObject.Invoke("setCheckpointDir", directory);
         }
 
+        /// <summary>
+        /// Broadcast a read-only variable to the cluster, returning a Microsoft.Spark.Broadcast
+        /// object for reading it in distributed functions. The variable will be sent to each 
+        /// executor only once.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="value">Value/variable to be broadcast</param>
+        /// <returns>A Microsoft.Spark.Broadcast object</returns>
+        public Broadcast<T> Broadcast<T>(T value)
+        {
+            var localDir = (string)_jvmObject.Jvm.CallStaticJavaMethod(
+                "org.apache.spark.util.Utils",
+                "getLocalDir",
+                _conf);
+            TempDir = Path.Combine(localDir, "sparkdotnet");
```

Comment:
do we want `sparkdotnet` here? How does pyspark constructor this?

Reply:
pyspark adds a "pyspark" folder layer from the local directory, that's where I got this from. Please let me know if it makes sense, thanks!

Reply:
ok then it should be fine.